lib/utils/get-proxy-agent.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const getProxyForUrl = require('proxy-from-env').getProxyForUrl;
+var HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
this should use const:
```suggestion
const HttpsProxyAgent = require('https-proxy-agent');
```

Reply:
I think we should also lazy require this since it won't be used most of the time?

Reply:
👍 good point, definitely agree